agent/instrumentation/applicationinsights-web-2.3/build.gradle.kts

```diff
@@ -12,12 +12,12 @@ muzzle {
 }
 
 val otelInstrumentationAlphaVersion: String by project
-val otelVersion: String by project
```

Comment:
can you rename otelVersion from dependencyManagement to otelSdkVersion? 

otelVersion seems confusing.

Reply:
sure, can I send a follow-up PR for that?

Reply:
np